main.c

```diff
@@ -633,6 +663,11 @@ int encode_resource_records(struct ResourceRecord* rr, uint8_t** buffer)
         for(i = 0; i < 16; ++i)
           put8bits(buffer, rr->rd_data.aaaa_record.addr[i]);
         break;
+      case TXT_Resource_RecordType:
+        put8bits(buffer, rr->rd_data.txt_record.txt_data_len);
+        for (i = 0; i < rr->rd_data.txt_record.txt_data_len; i++)
```

Comment:
please remove the single space after `for`. While the code style of the project is not the finest, at least it should be consistent.